src/components/EditButton/EditButton.jsx

```diff
@@ -0,0 +1,52 @@
+import React, { Component } from 'react';
+import { Button } from 'reactstrap';
+import ModalWindow from '../ModalWindow/ModalWindow';
+import './EditButton.scss';
+
+class EditButton extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      isOpen: false,
+    };
+
+    this.openModalWindow = this.openModalWindow.bind(this);
+  }
+
+  toggleModalWindow = () => {
+    this.setState((prev) => ({ ...prev, isOpen: !prev.isOpen }));
+  };
+
+  openModalWindow() {
+    this.setState({
+      isOpen: true,
+    });
+  }
+
+  render() {
+    const { title, body } = this.props;
+    const { isOpen } = this.state;
+
+    return (
+      <>
+        <Button
+          className="EditButton__btn"
+          color="primary"
+          onClick={this.openModalWindow}
+        >
+          Edit
+        </Button>
+        <ModalWindow
+          className="EditButton__modal"
+          title={title}
+          isOpen={isOpen}
+          body={body}
```

Comment:
rename the `body` to something more meaningful
I would also *consider* using this.props.children instead of props.body ... but just a though. Don't like both approaches either